### PR TITLE
[low code connectors] add missing/mistyped values for declarative sendgrid + sentry

### PR DESCRIPTION
**Pull request by brianjlai**
## What
After running schema validation against our existing connectors, there were a few small things that got flagged. These should be fixed before we merge the validation code, otherwise we'll break our daily connector testing

## How
A few connectors were missing version, and we should be sending all query parameters as strings as per our type definition

### CODE REVIEW

File: airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml [hunk 1]
```diff
@@ -247,7 +248,7 @@ streams:
         $ref: "*ref(definitions.requester)"
         request_options_provider:
           request_parameters:
-            limit: 1000
+            limit: "1000"
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-sendgrid/source_sendgrid/sendgrid.yaml:251]: The alternative to this, is we change the type of `RequestInput` from `RequestInput = Union[str, Mapping[str, str]]` to `RequestInput = Union[str, Mapping[str, Any]]`. That would then make 1000 a valid input

**Review by girarda — APPROVED** [review]
lgtm assuming the acceptance tests pass

**Pull request merged**